data/json/items/ammo.json

```diff
@@ -370,8 +370,7 @@
     "flags": [ "TRADER_AVOID" ],
     "weight": "657 g",
     "volume": "250 ml",
-    "bashing": 7,
```

Comment:
Don't remove this, this is for melee and the other is for ranged.

Reply:
Fixed